src/test/regress/expected/multi_cluster_management.out

```diff
@@ -164,7 +164,7 @@ SELECT shardid, shardstate, nodename, nodeport FROM pg_dist_shard_placement WHER
 
 -- try to remove a node with only inactive placements and see that removal still fails
 SELECT master_remove_node('localhost', :worker_2_port); 
-ERROR:  you cannot remove a node which has shard placements
+ERROR:  you cannot remove the primary node of a group which has shard placements
```

Comment:
maybe "primary node of a group" -> "primary of a node group", since it's not entirely clear what "group" refers to.